tests/modeling/test_roi_heads.py

```diff
@@ -131,6 +137,36 @@ def test_rroi_heads(self):
             ),
         )
 
+    @unittest.skipIf(TORCH_VERSION < (1, 7), "Insufficient pytorch version")
+    def test_mask_head_scriptability(self):
+        input_shape = ShapeSpec(channels=1024)
+        mask_features = torch.randn(4, 1024, 14, 14)
+
+        image_shapes = [(10, 10), (15, 15)]
+        pred_instance0 = Instances(image_shapes[0])
+        pred_classes0 = torch.tensor([1, 2, 3], dtype=torch.int64)
+        pred_instance0.pred_classes = pred_classes0
+        pred_instance1 = Instances(image_shapes[1])
+        pred_classes1 = torch.tensor([4], dtype=torch.int64)
+        pred_instance1.pred_classes = pred_classes1
+
+        mask_head = MaskRCNNConvUpsampleHead(
+            input_shape, num_classes=80, conv_dims=[256, 256]
+        ).eval()
+        origin_outputs = mask_head(mask_features, [pred_instance0, pred_instance1])
```

Comment:
Do we need to remove `pred_masks`? It is in-place added in `MaskRCNNConvUpsampleHead`. Or we can use `deepcopy` to avoid it.

Reply:
good point! deepcopy of the input would be good here

Reply:
done, I also use `deepcopy` in `from_instances` to avoid unexpected changes for origin instance caused by new instance.